pages/[user]/[type].tsx

```diff
@@ -153,98 +153,185 @@ export default function Type(props) {
+                    : "sm:w-1/2 sm:pl-4")
+                }
+              >
+                <div className="flex text-gray-600 font-light text-xl mb-4 ml-2">
+                  <span className="w-1/2">
+                    {dayjs().month(selectedMonth).format("MMMM YYYY")}
+                  </span>
+                  <div className="w-1/2 text-right">
+                    <button
+                      onClick={decrementMonth}
+                      className={
+                        "mr-4 " +
+                        (selectedMonth < parseInt(dayjs().format("MM")) &&
+                          "text-gray-400")
+                      }
+                      disabled={selectedMonth < parseInt(dayjs().format("MM"))}
+                    >
+                      <ChevronLeftIcon className="w-5 h-5" />
+                    </button>
+                    <button onClick={incrementMonth}>
+                      <ChevronRightIcon className="w-5 h-5" />
+                    </button>
+                  </div>
                 </div>
-            </main>
-        </div>
+                <div className="grid grid-cols-7 gap-y-4 text-center">
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Sun
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Mon
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Tue
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Wed
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Thu
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Fri
+                  </div>
+                  <div className="uppercase text-gray-400 text-xs tracking-widest">
+                    Sat
+                  </div>
+                  {calendar}
+                </div>
+              </div>
+              {selectedDate && (
+                <div className="sm:pl-4 mt-8 sm:mt-0 text-center sm:w-1/3  md:max-h-97 overflow-y-auto">
+                  <div className="text-gray-600 font-light text-xl mb-4 text-left">
+                    <span className="w-1/2">
+                      {dayjs(selectedDate).format("dddd DD MMMM YYYY")}
+                    </span>
+                  </div>
+                  {!loading ? availableTimes : <div className="loader"></div>}
+                </div>
+              )}
+            </div>
+          </div>
+          {/* note(peer): 
+            you can remove calendso branding here, but we'd also appreciate it, if you don't <3 
+          */}
+          <div className="text-xs text-right pt-1">
+            <Link href="/pricing">
```

Comment:
/pricing doesn't go anywhere, it should go to calendso.com instead
```suggestion
            <Link href="https://calendso.com">
```